Cargo.toml

```diff
@@ -11,6 +11,6 @@ repository = "https://github.com/Eonm/nl80211"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-buffering = { version = "0.3.2",  features = ["copy"] }
+byteorder = "1.4.3"
+macaddr = "1.0.1"
 neli = "0.4.3-r1"
```

Comment:
I think specifying the patch version doesn't limit you to that patch version - Cargo selects `0.4.4`

I did attempt to update to `0.5.x` but it was non-trivial. Do you thing it's worth spending time on for this PR?

Reply:
Ah, true. Defaults to caret so it will go up to `0.4.x`

And no, for this PR I would avoid migrating to `0.5.x` because as you said, it is non-trivial. I would create a jira so we don't lose track of this.